docs/components/codemirror.js

```diff
@@ -51,6 +51,15 @@ export default {
       CM: null
     }
   },
+  computed: {
+    componentData() {
```

Comment:
Why do we move this static data to computed properties?

Reply:
Just to keep the render function succinct and short 'n sweet.

Plus saves the object from having to be "built" on each render update.

Reply:
Also trying to make codemirror an async component, to reduce vendor bundle size:

![image](https://user-images.githubusercontent.com/2781561/61272744-32b5bd80-a77e-11e9-9d42-b0eac156d2d0.png)

Since it is only needed for the playground

Reply:
Ok good.